src/thing/Portal.cpp

```diff
@@ -0,0 +1,112 @@
+#include "thing/Portal.h"
```

Comment:
should we use https://github.com/tzapu/WiFiManager instead or rolling something on our own?